drizzlepac/alignimages.py

```diff
@@ -896,7 +930,7 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
 
         # Execute checks
         nmatches_check = False
-        if num_xmatches > 4:
+        if num_xmatches > 5:
```

Comment:
Please change the hard-coded "5" to a module level constant with a descriptive name and ensure the MIN_CROSS_MATCHES reconciles with this new constant.  I would also ask here if MIN_CROSS_MATCHES would work for both checks (line 924) and line 933 (if num_xmatches >= MIN_CROSS_MATCHES)?